R/service-navigation.R

```diff
@@ -0,0 +1,138 @@
+#' Service navigation
+#'
+#' @description
+#' Service navigation component consistent with the
+#' [GDS service navigation](https://design-system.service.gov.uk/components/service-navigation/)
+#' (pre-June 2025).
+#'
```

Comment:
We should had `service_name` as an optional argument into this function like the examples in the docs - https://design-system.service.gov.uk/components/service-navigation/